saleor/core/exceptions.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, message=None, *, permissions: Optional[Iterable[Enum]] = None
             if permissions:
                 permission_list = ", ".join(p.name for p in permissions)
                 message = (
-                    f"You need one of the following permissions: {permission_list}"
+                    f"To access this path, you need following permissions: {permission_list}"
```

Comment:
Grammar fix:
```suggestion
                    f"To access this path, you need the following permissions: {permission_list}"
```
Not sure if it doesn't exceed the line limit after this change, so the string would need to be split into two lines.

Reply:
All aforementioned remarks have been taken into account and corrections have been implemented.